NEWS.md

```diff
@@ -108,7 +110,10 @@ Other improvements and bug fixes:
 
 * `use_github()` tries harder but also fails earlier, with more informative
   messages, making it less likely to leave the repo partially configured (#221).
-  
+
+* `use_github()` now removes newline `\n` characters when pushing that can cause 
+  `gh::gh` fail (#493, @muschellij2).  
+
```

Comment:
This shouldn't be in this PR. It's conceivable you'd be happiest to close this PR and ... get synced up with usethis master, make a fresh branch, and make a new PR. There's a lot going on here w.r.t the git history.

Reply:
This has been removed.